assets/js/admin/admin.js

```diff
@@ -221,4 +221,61 @@
 		$(this).append(video);
 
 	});
+
+	// Toggles edit state.
+	function toggleEditTitle( e ) {
+		e.preventDefault();
+
+		var $ele = $( '#edit-form-name' ),
+			$input_title = $ele.siblings( '#evf-edit-form-name' );
```

Comment:
Need a blank line after all the variable declaration and rather than declaring `$ele` please prefer `$el` as it sounds better. Below snippet might be helpful xD

```
var $el          = $( '#edit-form-name' ),
	$input_title = $el.siblings( '#evf-edit-form-name' );
```

Reply:
This has been fixed :)